.github/workflows/label.yml

```diff
@@ -0,0 +1,61 @@
+name: Labeller
+on:
+  pull_request_target:
+    types: closed
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: .github/scripts
+      - name: Add release labels on merge
+        run: |
+          PR_NUMBER="${{ github.event.pull_request.number }}"
```

Comment:
Can you extract this to a dedicated `.sh` script? That way we can easily run locally to test.

Reply:
Done. Much nicer this way.